cli/tests/test_parallelize_command.rs

```diff
@@ -467,16 +501,28 @@ fn test_parallelize_failure_multiple_heads_with_different_children() {
     ◉  000000000000
     "###);
 
-    insta::assert_snapshot!(
-    test_env.jj_cmd_failure(
+    test_env.jj_cmd_ok(
         &workspace_path,
         &[
             "parallelize",
             "description(1)::description(2)",
             "description(a)::description(b)",
         ],
-    ),@r###"
-    Error: All heads of the target revisions must have the same children.
+    );
+    insta::assert_snapshot!(get_log_output(&test_env, &workspace_path), @r###"
```

Comment:
Very cool! nit: remove "failure" from the test name.

Reply:
Done.